docs/conf.py

```diff
@@ -283,3 +279,34 @@ def setup(app):
 # .htaccess) here, relative to this directory. These files are copied
 # directly to the root of the documentation.
 html_extra_path = ['robots.txt']
+
+
+def rstjinja(app, docname, source):
+    """Render pages as a jinja template to hide/show dev docs. """
+    # Make sure we're outputting HTML
+    if app.builder.format != 'html':
+        return
+    files_to_render = ["index", "install"]
+    if docname in files_to_render:
+        print(f"Jinja rendering {docname}")
+        rendered = app.builder.templates.render_string(
+            source[0], app.config.html_context)
+        source[0] = rendered
+
+
+def setup(app):
+    if sphinx_gallery is None:
+        msg = ('The sphinx_gallery extension is not installed, so the '
+               'gallery will not be built.  You will probably see '
+               'additional warnings about undefined references due '
+               'to this.')
+        try:
+            app.warn(msg)
+        except AttributeError:
+            # Sphinx 1.6+
+            from sphinx.util import logging
+            logger = logging.getLogger(__name__)
+            logger.warning(msg)
+
+    # Generate the page from Jinja template
+    app.connect("source-read", rstjinja)
```

Comment:
Nice, I didn't even realize it was possible to do this.

Reply:
Me neither. Thanks to @Cadair for letting me copy-paste from SunPy!